root_VS2010/programs/C#/10_Build_WebApp_sample.bat

```diff
@@ -13,7 +13,7 @@ set ORG_PATH=%PATH%
 @rem --------------------------------------------------
 @rem Get the path to the executable file.
 @rem --------------------------------------------------
-set CURRENT_DIR=%~dp0
+set CURRENT_DIR="%~dp0"
```

Comment:
@daisukenishino 

>https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/23#issue-123412442

I understood the changing the folder structure.
But, you do not specify anywhere that you continue modification of root_VS2010.

Therefore, I do not understand whether root_VS2010 remains as "read-only" or not.
And, "Future merge path" seems to contradict "Requirement".